app/services/ccms/payload_generators/entity_attributes_generator.rb

```diff
@@ -18,7 +18,8 @@ def initialize(requestor, xml, entity_name, options)
         @entity_name = entity_name
         @requestor = requestor
         @options = options
-        @attr_config = ccms_attribute_keys[@entity_name]
+        ccms_attributes_hash = ActiveSupport::HashWithIndifferentAccess.new(ccms_attribute_keys)
+        @attr_config = ccms_attributes_hash[@entity_name]
```

Comment:
- [ ] Find out why in ActiveSupport::HashWithIndifferentAccess.new helped resolve an issue in testing where it returned nil
- [ ] What is the cause of ccms_attribute_keys having keys that are strings and failing tests.